src/main/java/Commands/AddCommand.java

```diff
@@ -0,0 +1,32 @@
+package Commands;
```

Comment:
Might want to follow the coding standards, package names should be in all lowercase to avoid conflict in the names of classes or interfaces.